app/models/ems_event.rb

```diff
@@ -223,7 +223,7 @@ def get_target(target_type)
     target_type = "src_vm_or_template"  if target_type == "src_vm"
     target_type = "dest_vm_or_template" if target_type == "dest_vm"
     target_type = "middleware_server"   if event.event_type == "hawkular_alert"
-    target_type = "container_node"      if event.event_type == "datawarehouse_alert"
+    target_type = "target"              if event.event_type == "datawarehouse_alert"
```

Comment:
@moolitayer the `datawarehouse_alert` still makes sense as name for the `event_type`?
Or you're not changing it because it's not worth (too many changes, maybe a migration, etc.)?

Reply:
Here I'm changing only what is user exposed, defiantly worth changing the event name too, and there is a PR up for that, but it's not a high priority.

Reply:
`target_type = "target"` is extremely vague/generic.  Could we have some more specific targets set by the event catcher?

Reply:
@agrare I'm not sure how to do that:
- The Prometheus Alerts integration needs to support two types of targets (ext + container_node)
- During alert evaluation [1] Automate calls `get_target` to find the ems target[2]

The catcher already sets the generic target of the EmsEvent (`target` - I'm not sure why automate isn't using that). We could move the switch to [1] but it seems like that would move it away from handling of other types.

[1] https://github.com/manageiq/manageiq/blob/266cde6b9a68459c05c934192f2f2e713b33496d/lib/ems_event_helper.rb#L33
[2] https://github.com/manageiq/manageiq/blob/266cde6b9a68459c05c934192f2f2e713b33496d/app/models/ems_event/automate.rb#L108

Reply:
Hard-coding the target_type to `src_vm` there seems like a bad idea, @lfu @gtanzillo you guys had some changes to this area, what do you think of `policy(@event.target_type, @event.event_type)`?

Reply:
@moolitayer @agrare ```target_type = "target"``` Here ```target``` means the target column in event_streams table which is the object the event is related to. 
I guess this target of prometheus events could be either ```ems``` or ```container_node```. Is it right?

Reply:
Right @lfu, does `get_target` in automate have a different meaning?